.github/workflows/benchmark_large.yml

```diff
@@ -23,6 +23,7 @@ on:
         # Please keep this default value in sync with the jobs.build_e2e_test_artifacts.with.shard-count field below
         default: a2-highgpu-1g=1,c2-standard-60=2,default=1
         type: string
+  pull_request:
```

Comment:
revert before merge
also, you can trigger with workflow_dispatch if you push to a branch in this repo (not a fork) and use the "run workflow" button on https://github.com/iree-org/iree/actions/workflows/benchmark_large.yml